miracle/core/extractors.py

```diff
@@ -0,0 +1,148 @@
+import os
+import pyunpack as p
```

Comment:
would prefer explicit package imports so we don't have to remember the shorthand they were aliased as, same with `import collections as c`